mods/ra/maps/sarin-gas-1-crackdown/crackdown.lua

```diff
@@ -41,28 +41,16 @@ SetupTriggers = function()
 	Trigger.OnAllKilled(ConvoyTrucks, function()
 		greece.MarkCompletedObjective(objDestroyAllTrucks)
 	end)
-
-	Trigger.OnEnteredFootprint({ TruckEscapeCenter.Location }, function(actor, triggerlose1)
-		if actor.Owner == ussr and actor.Type == "truk" then
-			Trigger.RemoveProximityTrigger(triggerlose1)
-			actor.Destroy()
-			greece.MarkFailedObjective(objDestroyAllTrucks)
-		end
-	end)
-
-	Trigger.OnEnteredFootprint({ EscapeNorth10.Location }, function(actor, triggerlose2)
-		if actor.Owner == ussr and actor.Type == "truk" then
-			Trigger.RemoveProximityTrigger(triggerlose2)
-			actor.Destroy()
-			greece.MarkFailedObjective(objDestroyAllTrucks)
-		end
-	end)
-
-	Trigger.OnEnteredFootprint({ EscapeSouth5.Location }, function(actor, triggerlose3)
+	
+	Trigger.OnEnteredFootprint({ TruckEscapeCenter.Location, EscapeNorth10.Location, EscapeSouth5.Location }, function(actor, trigger)
 		if actor.Owner == ussr and actor.Type == "truk" then
-			Trigger.RemoveProximityTrigger(triggerlose3)
-			actor.Destroy()
-			greece.MarkFailedObjective(objDestroyAllTrucks)
+			Trigger.OnIdle(actor, function ()
+				if actor.IsIdle then
```

Comment:
This does not make sense, if this function runs the actor is certainly idle.

Reply:
I was asleep from the looks of it. Thanks!